### DISCUSSION TITLE
How do I get access to the Latent Gradient of a model

### DISCUSSION THREAD

**Discussion by Rami-Ismael** [larq/compute-engine#714] [Q&A] [answered]
For my project I need to get access to the gradient of the binary neural network.  I was hoping somone help me. I can get the latent gradient, but not the binary gradient

**Comment by CNugteren** [comment, marked as answer]
This is not something for the Larq Compute Engine, which is an inference engine, but more something for training Binarized Neural Networks. So I suggest to ask this question on the Larq project, which you seem to already have done [here](https://github.com/larq/larq/discussions/754). Let's continue there.
